src/ble/BUILD.gn

```diff
@@ -40,6 +40,10 @@ buildconfig_header("ble_buildconfig") {
     defines +=
         [ "BLE_PLATFORM_CONFIG_INCLUDE=${chip_ble_platform_config_include}" ]
   }
+  if (chip_device_platform == "esp32") {
+    defines +=
+        [ "BLE_PLATFORM_CONFIG_INCLUDE=${chip_ble_platform_config_include}" ]
+  }
```

Comment:
Is this block intended to be added? It looks like BLE_PLATFORM_CONFIG_INCLUDE is set twice here.

Reply:
Ah, the upper 'if' condition had a typo, fixed.